src/main/webapp/chat.js

```diff
@@ -0,0 +1,32 @@
+async function addToChat() {
+  let chatInput = document.getElementById("chat-input").value;
+  let data = {
+    message: chatInput,
+  };
+
+  let response = await fetch("/chat", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  });
+}
+
+function updateChat(data) {
+  let newMessage = document.createElement("li");
+  newMessage.appendChild(document.createTextNode(data.message));
+  let chat = document.getElementById("chat");
+  chat.appendChild(newMessage);
+}
+
+Pusher.logToConsole = false;
+
+var pusher = new Pusher("d15fbbe1c77552dc5097", {
```

Comment:
Could you create a constant for the CLIENT_KEY here like you did in the Java code, that way it's clear what this string represents?